### DISCUSSION TITLE
support for Kawpow algo

### DISCUSSION THREAD

**Discussion by jims23211** [Lolliedieb/lolMiner-releases#668] [Ideas]
Is it on the road map to add support for the KawPow algorithm in the near future as the end of mining ETH is fast approaching and RVN seems to be one of the viable successors to it?  I ask because right now, the only algorithm that supports GCN 1.0 GPU's is lolminer and I would love to continue to see them used.

**Comment by grzech1991** [comment]
has something changed in this topic?

**Comment by odee2004** [comment]
+1 to this
